API.md

```diff
@@ -226,6 +226,8 @@ The `server.auth.strategy()` method requires the following strategy options:
   object will be merged with the Wreck request object used to call the token endpoint. Such an
   object can contain custom HTTP headers or TLS options (e.g.
   `{ agent: new Https.Agent({ cert: myClientCert, key: myClientKey}) }`).
+  To allow dynamically updating secret, this option can be passed as a *function* returning string
```

Comment:
We should mention that the fonction is called synchronously.

Reply:
I think _function returning string_ is equivalent to `() => string` in TypeScript, not `() => string | Promise<string>`. Though, we can just add `await` in `lib/oauth.js:267`

Reply:
I agree but we didn't express any type definition here. This was just to prevent people from using async function and then wonder why it does not work the way they intended to. I don't expect every user to look at the code. I'm in favor of leaving sync for now especially you don't seem to need it. We can always reassess later if someone needs it.